ArchiSteamFarm/Json/Steam.cs

```diff
@@ -467,33 +467,51 @@ internal sealed class InventoryResponse : NumberResponse {
 					LastAssetID = lastAssetID;
 				}
 			}
-#pragma warning restore IDE0051
+			#pragma warning restore IDE0051
 
-#pragma warning disable IDE0051
+			#pragma warning disable IDE0051
 			[JsonProperty(PropertyName = "more_items", Required = Required.DisallowNull)]
 			private byte MoreItemsNumber {
 				set => MoreItems = value > 0;
 			}
-#pragma warning restore IDE0051
+			#pragma warning restore IDE0051
 
 			[JsonConstructor]
 			private InventoryResponse() { }
 
 			internal sealed class Description {
-				[JsonExtensionData]
-				internal readonly ImmutableDictionary<string, JToken> AdditionalProperties;
-
 				[JsonProperty(PropertyName = "appid", Required = Required.Always)]
 				internal readonly uint AppID;
 
+				[JsonExtensionData]
+				internal ImmutableDictionary<string, JToken> AdditionalProperties;
+
 				internal ulong ClassID { get; private set; }
 				internal ulong InstanceID { get; private set; }
 				internal bool Marketable { get; private set; }
 				internal Asset.ERarity Rarity { get; private set; }
 				internal uint RealAppID { get; private set; }
+
+				[JsonProperty(PropertyName = "tags", Required = Required.DisallowNull)]
+				internal ImmutableHashSet<Tag> Tags {
```

Comment:
I'm not sure if it wouldn't be better if this was just a typical property, and `Type`, `Rarity` and `RealAppID` was generated on-demand with a getter from tags, instead of interpreted once here.

This way we wouldn't waste CPU doing that per each item, and it'll still be fast enough for those items we need to check on.